data/FilterList.json

```diff
@@ -9745,191 +9745,191 @@
   },
   {
     "id": 481,
-    "chatUrl": null,
-    "description": null,
-    "descriptionSourceUrl": null,
+    "chatUrl": "https://t.me/EnergizedProtection",
+    "description": "True Lite Hosts from Energized. Only 1MB in Size.",
+    "descriptionSourceUrl": "https://github.com/EnergizedProtection/block",
     "discontinuedDate": null,
-    "donateUrl": null,
+    "donateUrl": "shadmansaleh@codeian.com",
```

Comment:
`donateUrl` should really be a url to a webpage about donating to the list. It could be either a page on the website that talks about donating, or to something like a PayPal page directly. I'm not sure how somebody would donate via email.